### ISSUE TITLE
Lack of lang attribute

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#52]
**Pagina:**
[Rijkshuistijl Community Template](https://rijkshuisstijl-community-templates-git-102644-nl-design-system.vercel.app/developer-overheid/homepage)

**Probleembeschrijving:**
De webpagina mist een lang-attribuut.

**Gerelateerde WCAG:**
WCAG 2.4.2 (Page Titled)

**Referentieafbeelding:**
<img width="849" alt="browser tab showing the url due to a lack of title element in the head" src="https://github.com/user-attachments/assets/18409802-07e4-4e3a-99ad-9745cd48fefd" />
